irohad/main/impl/consensus_init.cpp

```diff
@@ -96,7 +96,8 @@ namespace iroha {
           std::shared_ptr<network::BlockLoader> block_loader,
           const shared_model::crypto::Keypair &keypair,
           std::chrono::milliseconds vote_delay_milliseconds,
-          std::chrono::milliseconds load_delay_milliseconds) {
+          std::chrono::milliseconds load_delay_milliseconds,
+          std::shared_ptr<consensus::ConsensusResultCache> consensus_result_cache) {
```

Comment:
Maybe add it before configuration parameters, since it is logically a dependency